include/magic_enum.hpp

```diff
@@ -705,7 +708,7 @@ template <typename E>
 template <auto V>
 [[nodiscard]] constexpr auto enum_name() noexcept -> detail::enable_if_enum_t<decltype(V), string_view> {
   constexpr string_view name = detail::enum_name_v<std::decay_t<decltype(V)>, V>;
-  static_assert(name.size() > 0, "Enum value does not have a name.");
+  static_assert(name.size() > 0, "magic_enum::enum_name enum value does not have a name.");
```

Comment:
[clang-tidy says](https://clang.llvm.org/extra/clang-tidy/checks/readability-container-size-empty.html) use `!name.empty()` instead of `name.size() > 0`

Reply:
fixed, thanks